frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -866,14 +867,14 @@ export class TileViewPage extends React.Component {
             <div className="co-catalog-page__heading text-capitalize">{activeCategory.label}</div>
             <div className="co-catalog-page__filter">
               <div>
-                <TextInput
+                <SearchInput
```

Comment:
This is a `[beta` component in PatternFly.](https://www.patternfly.org/v4/components/search-input).  Everyone ok with that?

Reply:
[Slack message from Nicole Thoen](https://patternfly.slack.com/archives/C4FM977N0/p1648653837199109?thread_ts=1648652713.505589&cid=C4FM977N0) on the PatternFly team:
```
Hello! We don’t have a target date as of yet for getting it out of beta, but I could definitely push that conversation along.

In general, whenever you have a question about using a beta component in product, I have two suggestions:

1.  Play with the component examples in the docs and determine whether or not the current implementation meets your needs. Open issues for any missing features, enhancement requests, or bugs.
2.  Check the [PatternFly Issues GitHub Project board](https://github.com/orgs/patternfly/projects/7/views/5) where we have created a view to categorize most issues by component. Browse any open issues for the beta component to determine how much more the beta component could evolve in the near future.

In the next few releases there will be a few changes refactored under the hood (which should not change the interactions/features - only the underlying class names and element groupings) https://github.com/patternfly/patternfly-react/issues/7121

Outside of the refactoring effort I’ve mentioned, there are not issues that look like they will introduce breaking changes, which means if I push it along, it could be a candidate for promotion soon. (edited)
```
Based on this, I am confident we can use `<SearchInput>` in this PR.